scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1740,7 +1740,7 @@ public void processDatagram(DatagramPacket datagram) {
 	 * @param router router address, {@code null}, if no router is used.
 	 * @since 2.5
 	 */
-	protected void processDatagram(DatagramPacket packet, InetSocketAddress router) {
+	protected void processDatagram(DatagramPacket packet, final InetSocketAddress router) {
```

Comment:
Please remove the added `final`s. I don't see, its required, or?

Reply:
You're right, I left them accidentally